src/compiler/scala/tools/nsc/ast/parser/Parsers.scala

```diff
@@ -654,6 +654,8 @@ self =>
     def isRawBar   = isRawIdent && in.name == raw.BAR
     def isRawIdent = in.token == IDENTIFIER
 
+    def isIdentExceptRaw(except: Name) = isIdent && !(isRawIdent && in.name == except)
```

Comment:
We neglected to say, `isIdent && !isRawBar` at 1971.

Once we'd said that, "raw oyster bar" could not be far behind.